README.md

```diff
@@ -1,57 +1,63 @@
 # Guide
 
-A website to teach Wagtail CMS to content editors, moderators and administrators.
+Guide is a website to help content editors, moderators, administrators, and other users learn how to user the Wagtail content management system (CMS).
 
-Wagtail guide brings you:
-- Tutorials, 
-- How-to, 
-- Reference 
-- Background information
+The Wagtail guide will ultimately include:
+- Tutorials 
+- How-to articles
+- Reference materials
+- Walkthroughs and visual learning materials
 
+# Table of Contents
 
-## Google Summer of Code
+- [Installation](#installation)
+- [Gitpod](#gitpod)
+- [Contributing](#contributing)
+- [Other Notes](#other-notes)
 
-This project is a part of Google Summer of Code 2022 under the organization [**Wagtail**](https://wagtail.org/). Find more details about the project [here](https://summerofcode.withgoogle.com/programs/2022/projects/7nMw2hTq).
+# Installation
 
+We assume that you have basic knowledge of Node/Yarn/Webpack and Python/Django/Wagtail in these instructions. We recommend you develop Wagtail Guide locally on your machine using venv.
 
-## Development
+#### Dependencies
+- Python 3.9
+- Git
+- Node 16.*
+- [Yarn](https://yarnpkg.com/)
 
-We assume that you have basic knowledge of Node/Yarn/Webpack and Python/Django/Wagtail.
+### Setting up Wagtail guide in a virtual environment
 
-### Frontend
+With [PIP](https://github.com/pypa/pip) installed, run:
```

Comment:
AFAIK Pip comes preinstalled with modern Python.

I think we can drop this line.

Reply:
Good catch.